_pages/en/about-us.html

```diff
@@ -5,7 +5,23 @@
 lang: en
 permalink: /about-us.html
 ---
+<div class="team-intro">
+<p>This website is a platform where we, IT Strategy, are sharing our plan for becoming a digital government, our progress, and all the tools, information and other resources needed to support this transformation across IITB and ESDC as a whole. 
+It is a place that gives us an opportunity to express our professional opinion and ideas.</p>
 
+<p>The ESDC IT Strategy team is working in the open and welcomes any feedback while work is still in progress.
+Contributing to work in progress is framing our learning experience and differs from formal committee reviews that most government employees are familiar with. 
+Working in the open is a culture and way of thinking.
+It is exposing our work and not keeping it in a silo.
+To keep our work more visible we chose GitHub as opposed to typical internal records management system because GitHub is available on the Internet and makes use of Git. 
```

Comment:
At the end, I would add "version control system" to clarify what Git is.

Reply:
I added like ...version control system (GIT).